analyzers/src/SonarAnalyzer.CSharp/Rules/PropertyToAutoProperty.cs

```diff
@@ -35,56 +35,52 @@ namespace SonarAnalyzer.Rules.CSharp
     [Rule(DiagnosticId)]
     public sealed class PropertyToAutoProperty : SonarDiagnosticAnalyzer
     {
-        internal const string DiagnosticId = "S2292";
+        private const string DiagnosticId = "S2292";
         private const string MessageFormat = "Make this an auto-implemented property and remove its backing field.";
+        private const int ReadWriteAccessorCount = 2;
```

Comment:
Can be const inside that single method that uses it. Or just accept the magic number, as  `Count` is really easy to understand.